codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/ApplicationTransport.java

```diff
@@ -83,6 +83,8 @@ public static ApplicationTransport createDefaultHttpApplicationTransport() {
                 .name("endpoint")
                 .type("string")
                 .documentation("Endpoint of the service")
+                .initializationCustomization("@endpoint = options.fetch(:endpoint, options[:stub_responses] "
```

Comment:
I don't like example.com as a default. It's not owned by us and we can expect it not to be set. I'd rather raise an error in initialize if endpoint is not set (preferred) or use localhost as a reasonable default.

Reply:
Sure - I'll switch it to localhost.  This only gets applied when stubbing is true so I don't think it matters much what it is.

Reply:
It could have a side affect, like if stub_responses isn't applied (send middleware swap?) or some other failure.